generated/installer/ibm-spectrum-scale-csi-operator-ocp-rhel.yaml

```diff
@@ -46,13 +46,13 @@ spec:
         - name: CSI_DRIVER_IMAGE
           value: quay.io/ibm-spectrum-scale-dev/ibm-spectrum-scale-csi-driver@sha256:fe72e11647adda33460df7ff9141f7fbd4040ea02da384a89738a9f87b4967e7
         - name: CSI_SNAPSHOTTER_IMAGE
-          value: registry.k8s.io/sig-storage/csi-snapshotter@sha256:2e04046334baf9be425bb0fa1d04c2d1720d770825eedbdbcdb10d430da4ad8c
+          value: registry.k8s.io/sig-storage/csi-snapshotter@sha256:b3e90b33781670ac050c22c9e88b9e876493dca248966b9da6f7a90cc412ab86
```

Comment:
@Jainbrt @shrutinipane  do we need this file `generated/installer/ibm-spectrum-scale-csi-operator-ocp-rhel.yaml` for 2.13 onwards ?